### PR TITLE
Fix URL for Hype Machine album premieres.

### PR DESCRIPTION
**Pull request by jaccarmac**
I noticed the URL of the [latest premiere](https://hypem.com/album/alostmen) changed.

Looking at the [list of albums](https://hypem.com/albums), it seems like all of them have changed in the same way.

Would it be best to change the name of the connector to "Hype Machine Albums"?

### CODE REVIEW

**Comment by alexesprit** [comment]
> Would it be best to change the name of the connector to "Hype Machine Albums"?

Sure!

File: src/core/connectors.js [hunk 1]
```diff
@@ -318,9 +318,9 @@ const connectors = [{
 	js: 'connectors/tidal.js',
 	id: 'tidal',
 }, {
-	label: 'Hype Machine Premieres',
+	label: 'Hype Machine Featured Albums',
 	matches: [
-		'*://hypem.com/premiere/*',
+		'*://hypem.com/album/*',
```
Comment (alexesprit) [inline src/core/connectors.js:323]: I think it should be updated too.

File: src/core/connectors.js [hunk 2]
```diff
@@ -318,9 +318,9 @@ const connectors = [{
...
 	],
 	js: 'connectors/hypem-premieres.js',
```
Comment (alexesprit) [inline src/core/connectors.js:325]: I think it should be updated too.

Comment (jaccarmac) [reply src/core/connectors.js:325]: Just the filename or the `id` as well? If `id` is changed, should I write a migration to keep options set for it?

Comment (alexesprit) [reply src/core/connectors.js]: No, I meant just the filename. ID should be kept as is for compatibility.

**Review by alexesprit — APPROVED** [review]

**Pull request merged**